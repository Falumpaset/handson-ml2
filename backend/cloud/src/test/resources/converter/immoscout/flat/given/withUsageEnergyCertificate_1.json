{
  "id": 1,
  "customer": {
    "name": "Fabian Beck Immobilien"
  },
  "user": {
    "profile": {
      "firstname": "Fabian",
      "name": "Beck"
    }
  },
  "prioset": {
    "data": {
      "wbs": true
    }
  },
  "externalId": 1,
  "data": {
    "name": "3 Zimmer in Sonsbeck",
    "size": 55.0,
    "attic": true,
    "floor": 1,
    "rooms": 3.0,
    "garden": true,
    "ground": "LAMINATE",
    "heater": "OVEN",
    "address": {
      "city": "Sonsbeck",
      "region": "Nordrhein-Westfalen",
      "street": "Haubachstra√üe",
      "country": "DE",
      "zipCode": "47665",
      "additional": null,
      "houseNumber": "44"
    },
    "contact": null,
    "seniors": false,
    "bailment": 100.0,
    "elevator": false,
    "flatType": "LOFT",
    "landArea": null,
    "basePrice": 400.0,
    "bathRooms": 2,
    "documents": [],
    "fireplace": true,
    "flatShare": false,
    "gardenUse": true,
    "houseType": null,
    "storeRoom": false,
    "externalId": null,
    "garageData": null,
    "gardenArea": null,
    "objectType": "FLAT",
    "tvSatCable": true,
    "attachments": [],
    "barrierFree": false,
    "bicycleRoom": true,
    "heatingCost": null,
    "kitchenette": false,
    "referenceId": null,
    "showAddress": true,
    "showContact": false,
    "washDryRoom": false,
    "basementSize": 10.0,
    "availableFrom": "morgen",
    "guestToilette": false,
    "parkingSpaces": [
      {
        "type": "UNDERGROUND_CAR_PARK",
        "count": 1,
        "price": 20.0
      }
    ],
    "serviceCharge": 20.0,
    "commercialData": null,
    "furnishingType": null,
    "numberOfFloors": 2,
    "previousTenant": null,
    "totalRentGross": 440.0,
    "temporaryLiving": null,
    "constructionYear": "2020",
    "historicBuilding": false,
    "numberOfBedrooms": null,
    "numberOfTerraces": null,
    "basementAvailable": null,
    "bathroomEquipment": {
      "bidet": false,
      "shower": false,
      "urinal": false,
      "window": false,
      "bathtub": false
    },
    "buildingCondition": "FIRST_TIME_USE",
    "energyCertificate": {
      "creationDate": "MAY_2014",
      "usageCertificate": {
        "energyConsumption": "100",
        "energyEfficiencyClass": "A_P",
        "includesHeatConsumption": false,
        "energyConsumptionParameter": null
      },
      "demandCertificate": null,
      "yearOfConstruction": 2020,
      "energyCertificateType": "USAGE_IDENTIFICATION",
      "primaryEnergyProvider": "SOLAR",
      "primaryEnergyConsumption": "100"
    },
    "lastRefurbishment": null,
    "numberOfBalconies": 1,
    "objectDescription": "liegt",
    "totalParkingPrice": 20.0,
    "balconyTerraceArea": null,
    "objectLocationText": "Liegt auf dem Boden",
    "heatingCostIncluded": true,
    "furnishingDescription": "liegt",
    "objectMiscellaneousText": "test",
    "washingMachineConnection": false,
    "parkingPriceIncludedToAdditionalCosts": false
  }
}