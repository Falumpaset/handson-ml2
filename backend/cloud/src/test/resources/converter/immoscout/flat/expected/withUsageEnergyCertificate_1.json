{
  "externalId": "1",
  "title": "3 Zimmer in Sonsbeck",
  "creationDate": null,
  "lastModificationDate": null,
  "address": {
    "street": "Haubachstraße",
    "houseNumber": "44",
    "postcode": "47665",
    "city": "Sonsbeck",
    "internationalCountryRegion": {
      "country": "DEU",
      "region": "Nordrhein-Westfalen"
    },
    "quarter": null,
    "wgs84Coordinate": null,
    "preciseHouseNumber": null,
    "geoHierarchy": null,
    "description": null
  },
  "apiSearchData": null,
  "realEstateState": "ACTIVE",
  "id": null,
  "groupNumber": null,
  "descriptionNote": "liegt",
  "furnishingNote": "liegt",
  "locationNote": "Liegt auf dem Boden",
  "otherNote": "test",
  "attachments": null,
  "referencePriceServiceCall": null,
  "referencePriceApiCall": null,
  "realEstateProjectId": null,
  "showAddress": true,
  "contact": null,
  "publishChannels": null,
  "apartmentType": "LOFT",
  "floor": 1,
  "lift": false,
  "assistedLiving": null,
  "energyCertificate": {
    "energyCertificateAvailability": "AVAILABLE",
    "energyCertificateCreationDate": "FROM_01_MAY_2014",
    "energyEfficiencyClass": "A+",
    "electricityConsumption": null,
    "heatingConsumption": null,
    "energyConsumptionElectricity": null,
    "energyConsumptionHeating": null,
    "thermalCharacteristicElectricity": null,
    "thermalCharacteristicHeating": null
  },
  "cellar": "YES",
  "handicappedAccessible": null,
  "numberOfParkingSpaces": 1,
  "condition": "FIRST_TIME_USE",
  "lastRefurbishment": null,
  "interiorQuality": null,
  "constructionYear": 2020,
  "constructionYearUnknown": null,
  "freeFrom": "morgen",
  "heatingType": null,
  "heatingTypeEnev2014": "STOVE_HEATING",
  "firingTypes": null,
  "energySourcesEnev2014": {
    "energySourceEnev2014": [
      "SOLAR_HEATING"
    ]
  },
  "buildingEnergyRatingType": "ENERGY_CONSUMPTION",
  "thermalCharacteristic": 100.0,
  "energyConsumptionContainsWarmWater": null,
  "numberOfFloors": null,
  "usableFloorSpace": null,
  "numberOfBedRooms": null,
  "numberOfBathRooms": 2,
  "guestToilet": "NOT_APPLICABLE",
  "parkingSpaceType": "UNDERGROUND_GARAGE",
  "baseRent": 400.0,
  "totalRent": 440.0,
  "serviceCharge": 20.0,
  "deposit": "100,00 €",
  "heatingCosts": 0.0,
  "heatingCostsInServiceCharge": "YES",
  "petsAllowed": null,
  "parkingSpacePrice": 20.0,
  "useAsFlatshareRoom": null,
  "price": null,
  "livingSpace": 55.0,
  "numberOfRooms": 3.0,
  "energyPerformanceCertificate": true,
  "builtInKitchen": false,
  "balcony": true,
  "certificateOfEligibilityNeeded": true,
  "garden": true,
  "courtage": {
    "hasCourtage": "NO",
    "courtage": null,
    "courtageNote": null
  }
}